plans/feat-1858-keymap-discoverability.md

```
@@ -0,0 +1,216 @@
+| `Send keys to the terminal` | 1 以上 | 1 |
+| `ターミナルにキー列を送る` | 1 以上 | 1 |
+| 新しい intro（en / ja） | 1 以上 | 1 / 1 |
+| 旧 intro `Shortcuts are off until you bind them` | 0 | 0 |
+| 旧 intro の `ショートカットは {configFile}` | 0 | 0 |
+
+### `origin/main` を取り込んで壊れた（そして直した）
+
+レビュー中に #1890（launch panel、#1867）が main に入り、**`KEYMAP_ACTIONS` に 10 個目
+`terminal-new-here` が増えた**。マージした結果、行数を数える 2 件が赤くなった:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)